simplemind/concepts.py

```
@@ -1,6 +1,17 @@
-class Context:
-    def __init__(self):
-        self.plugins = [kv, basic_memory]
+from pydantic import BaseModel
+from typing import Dict, Any
```

Ruff

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)